taxi_service/settings.py

```diff
@@ -44,6 +44,8 @@
     "django.contrib.staticfiles",
     "debug_toolbar",
     "taxi",
+    "crispy_forms",
+    "crispy_bootstrap4",
```

Comment:
The 'crispy_forms' and 'crispy_bootstrap4' apps are correctly added to the INSTALLED_APPS list. This is necessary for using crispy forms as required by the task description.